### DISCUSSION TITLE
Order latency data question.

### DISCUSSION THREAD

**Discussion by christlsh** [nkaz001/hftbacktest#167] [Q&A]
asset = (
    BacktestAsset()
        .data([
            'data/ethusdt_20221003.npz'
        ])
        .initial_snapshot('data/ethusdt_20221002_eod.npz')
        .linear_asset(1.0)
        .intp_order_latency([
            'latency/feed_latency_20221003.npz'
        ])
        .power_prob_queue_model(2.0)
        .no_partial_fill_exchange()
        .trading_value_fee_model(-0.00005, 0.0007)
        .tick_size(0.01)
        .lot_size(0.001)
        .roi_lb(0.0)
        .roi_ub(3000.0)
        .last_trades_capacity(10000)
)

In the example market making strat, there is a file of latency in .intp_order_latency. I checked the code and it says one could generate it by interpolation or collect the real order latency. I'm wondering can any one provide a sample file for my reference? I would like to know the format and entries of the file. THX.

**Comment by nkaz001** [comment]
Please see https://hftbacktest.readthedocs.io/en/py-v2.1.0/tutorials/Order%20Latency%20Data.html
